### ISSUE TITLE
Crash while using an editor

### ISSUE THREAD

**Issue by rayesdamio** [4ian/GDevelop#3746]
I attempted to create a blank project and then the editor encountered a problem.

## Error stack (don't write anything here)
```
Error: Could not call remote method 'getPath'. Check that the method signature is correct. Underlying error: Failed to get 'documents' path
Underlying stack: Error: Failed to get 'documents' path
    at electron/js2c/browser_init.js:6745:67
    at IpcMainImpl.<anonymous> (electron/js2c/browser_init.js:6610:27)
    at IpcMainImpl.emit (events.js:210:5)
    at WebContents.<anonymous> (electron/js2c/browser_init.js:3873:23)
    at WebContents.emit (events.js:210:5)

    at electron/js2c/browser_init.js:6748:21
    at IpcMainImpl.<anonymous> (electron/js2c/browser_init.js:6610:27)
    at IpcMainImpl.emit (events.js:210:5)
    at WebContents.<anonymous> (electron/js2c/browser_init.js:3873:23)
    at WebContents.emit (events.js:210:5)
```

## Component stack (don't write anything here)
```

    in Unknown
    in t
    in ForwardRef
    in ForwardRef
    in t
    in Unknown
    in Unknown
    in div
    in n
    in div
    in Unknown
    in n
    in u
    in f
    in p
    in p
    in m
    in m
    in s
    in n
    in t
    in pe
    in n
    in Unknown
    in v
    in t
    in m
    in o
    in n
    in n
    in DragDropContext(n)
    in n
    in u
```

## Other details
* IDE version: 5.0.128-60088ebafd094b00260d3ec7f199186160317b02
* Arch: x64,
* Platform Name: win32,
* System Version: 10.0.19044,
* User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) GDevelop5/5.0.128 Chrome/80.0.3987.165 Electron/8.2.5 Safari/537.36,

**Comment by Oxey405** [comment]
Hello @rayesdamio ! I'm not a "member" of the GDevelop team but I'm asking those questions to be sure ;
- Can you verify that the directory you wanted your project to be saved in is writable by you ?
- If no can you run GDevelop with administrator permissions
- Can you make sure the directory entered exist ?
- Have you tried to uninstall and reinstall GDevelop ?
- Can you send some screenshots of your GDevelop configuration (in file > Preferences)
- Are you running on a nightly build ?
If you can answer to all these questions maybe someone can find a fix for this bug :)

**Comment by Bouh** [comment]
This bug has been addressed in 5.0.129.
If you can use the latest version it will help you ;)
